### PR TITLE
User model dependent

### PR DESCRIPTION
**Pull request by bmahmoudyan**
in this branch, I create ProfileImage, SocialLink, and ActivationCode models
also, fix bugs attribute error at User model

### CODE REVIEW

File: src/user/models/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
 from .user import User
+from .profile_image import ProfileImage
+from .social_link import SocialLink
+from .activation_code import ActivationCode
```
Comment (simpleasnothing) [inline src/user/models/__init__.py:4]: lint error
add a blank line to the end of this file to fix this issue

Comment (bmahmoudyan) [reply src/user/models/__init__.py]: I just add a new blank line to the end of the file

File: src/user/models/profile_image.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        User,
+        verbose_name=_("Avatar"),
+        on_delete=models.CASCADE,
+        related_name="profile_image"
+    )
+
+    image = models.ImageField(
+        verbose_name=_("Image"),
+        upload_to=get_file_path,
+        max_length=128,
```
Comment (simpleasnothing) [inline src/user/models/profile_image.py:31]: remove max_length

Comment (bmahmoudyan) [reply src/user/models/profile_image.py]: I deleted it

**Pull request merged**